committee-charters/partnerships-charter.md

```diff
@@ -6,44 +6,43 @@ _Why does this committee exist?_
 
 The Partnership committee exists to:
 
-* **Build, manage, and maintain mutually beneficial relationships** with external partners through a collaborative approach and effective coordination within the DAO.
-* **Identify and evaluate opportunities** by actively conducting market research, evaluating potential partners, and assessing strategic fit and feasibility.
-* **Structure partnerships** to establish a strong foundation for successful collaborations. This includes defining clear partnership objectives, allocating roles and responsibilities, establishing contractual agreements, outlining deliverables and milestones, facilitating communication and collaboration, and monitoring and evaluating progress.
+- **Build, manage, and maintain mutually beneficial relationships** with external partners through a collaborative approach and effective coordination within the DAO.
+- **Identify and evaluate opportunities** by actively conducting market research, evaluating potential partners, and assessing strategic fit and feasibility.
+- **Structure partnerships** to establish a strong foundation for successful collaborations. This includes defining clear partnership objectives, allocating roles and responsibilities, establishing contractual agreements, outlining deliverables and milestones, facilitating communication and collaboration, and monitoring and evaluating progress.
 
 ### Members
 
-* Nico Valencia 🧢
-* Katy Jeremko
-* Roy Rivera
+- Nico Valencia 🧢
+- Katy Jeremko
+- Roy Rivera
```

Comment:
I believe the S10 town hall slides had my name on them, which was a surprise to me. Just want to confirm I'm not expected to be on this committee?

Reply:
Roy should still be the other member on Partnerships (helping with LatAm expansion). Good catch! @nicovalencia can you update this one with the PR?